src/App/Header/index.tsx

```diff
@@ -20,6 +20,8 @@ export default function Header({ links, classes }: Props) {
             const isIntersecting = e.intersectionRatio < 1;
             e.target.classList.toggle("bg-white", isIntersecting);
             e.target.classList.toggle("shadow-lg", isIntersecting);
+            //remove extra px-4 in homepage
```

Comment:
Nit: Is this this a TODO or an explaination of what happened? We can prob remove is comment, if the latter.
```suggestion
```